model_path: "models/model.onnx"
num_inputs: 45
num_outputs: 12
